app/scripts/modules/kubernetes/src/instance/details/details.controller.js

```diff
@@ -221,14 +221,14 @@ module.exports = angular.module('spinnaker.instance.detail.kubernetes.controller
     this.deregisterInstanceFromLoadBalancer = function deregisterInstanceFromLoadBalancer() {
       var instance = $scope.instance;
       var loadBalancerNames = instance.loadBalancers.join(' and ');
-
+      
```

Comment:
Extra whitespace?

Reply:
I saw that and tried to get rid of it a few times but couldn't. when i remove it, it's still a deleted line.